x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -297,7 +295,8 @@ const UnstyledProcessEventDot = React.memo(
      */
     return (
       <div
-        data-test-subj={'resolverNode'}
+        data-test-subj="resolver:node"
```

Comment:
future work: make these IDs unique to each document location ID so we can test 2 resolvers at once.